src/script/cpp_api/s_server.cpp

```diff
@@ -59,6 +59,32 @@ bool ScriptApiServer::getAuth(const std::string &playername,
 	return true;
 }
 
+bool ScriptApiServer::getAuthLastLogin(const std::string &playername,
```

Comment:
`ScriptApiServer::getAuth(playername, nullptr, nullptr)`
could be re-used instead of adding a new function. The auth handler must return the other fields any way and checking those doesn't take much time.

Reply:
Thanks for the tip, Wouldn't adding another argument to the declaration of `getAuth` require that all other calls to that function elsewhere pass NULL? I was mainly trying to avoid changing any more files than necessary. But I can gladly do it this way if that's what you'd prefer.

Reply:
You can set the default argument value in the header file, then existing calls won't have be edited.